docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -357,38 +239,47 @@ Config:
 role_attribute_path = contains(info.roles[*], 'admin') && 'Admin' || contains(info.roles[*], 'editor') && 'Editor' || 'Viewer'
 ```
 
-#### Map server administrator privileges
-
-> Available in Grafana v9.2 and later versions.
+#### Map server administrator role
 
-If the application role received by Grafana is `GrafanaAdmin`, Grafana grants the user server administrator privileges.  
-This is useful if you want to grant server administrator privileges to a subset of users.  
-Grafana also assigns the user the `Admin` role of the default organization.
+In the following example, the user is granted the Grafana server administrator role.
 
-The setting `allow_assign_grafana_admin` under `[auth.generic_oauth]` must be set to `true` for this to work.  
-If the setting is set to `false`, the user is assigned the role of `Admin` of the default organization, but not server administrator privileges.
+Payload:
 
-```ini
-allow_assign_grafana_admin = true
+```json
+{
+    ...
+    "info": {
+        ...
+        "roles": [
+            "admin",
+        ],
+        ...
+    },
+    ...
+}
 ```
 
-Example:
+Config:
 
 ```ini
 role_attribute_path = contains(info.roles[*], 'admin') && 'GrafanaAdmin' || contains(info.roles[*], 'editor') && 'Editor' || 'Viewer'
+allow_assign_grafana_admin = true
 ```
 
-## Team synchronization
+## Configure team synchronization
 
-> Available in Grafana Enterprise v8.1 and later versions.
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise" >}}) and [Grafana Cloud](/docs/grafana-cloud/).
 
-With Team Sync you can map your Generic OAuth groups to teams in Grafana so that the users are automatically added to the correct teams.
+With Team Sync you can map your generic OAuth2 groups to teams in Grafana so that the users are automatically added to the correct teams.
```

Comment:
```suggestion
By using Team Sync, you can link your OAuth2 groups to teams within Grafana. This will automatically assign users to their appropriate teams.
```